test/fixup-svg-string.js

```diff
@@ -27,3 +27,18 @@ test('fixupSvgString should make parsing fixtures not throw', t => {
     });
     t.end();
 });
+
+test('fixupSvgString should correct invalid mime type', t => {
```

Comment:
might want to add a negated test like:

Does not manipulate "how to mime type wrong" svg text attribute where you test that it doesnt chage:
```
<text>data:img/png is not a mime type</text>
```